LeftNavBarExample/LeftNavBarLibrary/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'com.android.tools.build:gradle:2.2.1'
```

Comment:
Ok, followed your suggestion @mr-archano and moved this demo into a independent project: dec78a29bfe68f851b7a63c06d1dc7fece30a882